src/main/java/org/variantsync/diffdetective/variation/tree/VariationTreeNode.java

```diff
@@ -308,6 +308,27 @@ public static VariationTreeNode fromID(int id, List<String> label) {
         );
     }
 
+    public VariationTreeNode deepCopy(final Map<VariationTreeNode, VariationTreeNode> oldToNew) {
```

Comment:
Document the purpose of oldToNew. Couldn't you just generalize `VariationNode.toVariationTree`? A `deepCopy` alias would still make sense, though.

Reply:
Is generalized now. Generalization required an abstract method `shallowCopy` to create new nodes which is dependent on subclass.

Reply:
I though of something different and changed to my liking :)